app/models/admin_set.rb

```diff
@@ -1,5 +1,7 @@
 # There is an interplay between an AdminSet and a PermissionTemplate.
 # @see Hyrax::AdminSetBehavior for further discussion
 class AdminSet < ActiveFedora::Base
+  DEFAULT_ID = 'admin_set/default'.freeze
```

Comment:
OK, thanks, @tdonohue. I agree that we should stick with the constant for now, and move it to config IFF someone shares a use case that requires this.